### DISCUSSION TITLE
[solved] Unable to release a flow

### DISCUSSION THREAD

**Discussion by viko42** [postmanlabs/postman-flows#425] [Defect]
Hello Guys!

I tried to release some flows be I'm facing a problem.
```Connecting to remote session. Attempt #1/2/3/4/5```

<img width="458" alt="Screenshot 2022-12-09 at 17 42 45" src="https://user-images.githubusercontent.com/18552774/206751490-0144247b-140c-4c4e-b207-5eec32321a08.png">

Postman version: Version 10.5.8
I tried on my old flows, and brand new flows. Same error.
Do you guys have any ideas?

Article: https://learning.postman.com/docs/postman-flows/running-flows-on-the-cloud/webhooks/#create-a-release

Solved with:

Updating to 10.6.x
Thanks to [jon-flowers](https://github.com/jon-flowers)

**Comment by askherconsulting** [comment]
I am seeing this too.

**Comment by jon-flowers** [comment]
This functionality was significantly changed in the most recent (10.6) release.  See my comment here: https://github.com/postmanlabs/postman-app-support/issues/11518#issuecomment-1353396318
